apps/condo/domains/news/schema/NewsItem.js

```diff
@@ -136,10 +136,40 @@ const NewsItem = new GQLListSchema('NewsItem', {
         },
 
         sendAt: {
```

Comment:
what will happen with this value if user decides to click "No I do not want to send this newsItem"

Also, I think we should reuse 15 seconds from frontned part of our app. Otherwise we will change 15 to 30 in one place. and forget to change in other place

Reply:
>  if user decides to click "No I do not want to send this newsItem"

This news item will be deleted (using `deletedAt`)

Reply:
> Also, I think we should reuse 15 seconds from frontned part of our app. Otherwise we will change 15 to 30 in one place. and forget to change in other place

Fixed